Client/SafeBrowsing/GSafeBrowsingCache.swift

```diff
@@ -0,0 +1,123 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+private struct ThreatCache: Hashable {
+    var time: TimeInterval
```

Comment:
Are we storing the expiry datetime or the number of seconds to cache the entry? If this is the former, I would suggest renaming the variable to "expiry" to make it obvious.

Reply:
Fixed.